static/js/entry/faculty_carousel.js

```diff
@@ -7,9 +7,14 @@ import ReactDOM from 'react-dom';
 import FacultyCarousel from '../components/FacultyCarousel';
 
 const carouselDiv = document.querySelector('#faculty-carousel');
-if (carouselDiv !== null) {
+let facultyList = null;
```

Comment:
It looks like `FacultyCarousel` is expecting this to not be null. Can you default it to `[]` instead?